docs/usage.md

```diff
@@ -63,38 +70,73 @@ The **HPE 3PAR Docker Volume Plug-in** supports several optional parameters that
 
 - **qos-name** -- name of existing VVset on the HPE 3PAR where QoS rules are applied.(**introduced in plugin version 2.1**)
 
+- **replicationGroup** -- name of an existing remote copy group on the HPE 3PAR.(**introduced in plugin version 3.0**)
+
+- **fsOwner** -- user ID and group ID that should own root directory of file system.(**introduced in plugin version 3.0**)
+
+- **fsMode** -- mode of the root directory of file system to be specified as octal number.(**introduced in plugin version 3.0**)
+
+- **backend** -- backend to be used for the volume creation.(**introduced in plugin version 3.0**)
+
```

Comment:
Can we have a general help for getting the help options
`docker volume create -d hpe -o help` and
`docker volume create -d hpe -o help=backends`